Makefile

```diff
@@ -84,12 +87,7 @@ test-integration: ## Run integration tests
 .PHONY: test-e2e
 test-e2e: ## Run e2e tests
 	PULL_POLICY=IfNotPresent $(MAKE) docker-build
-	cd $(TEST_E2E_DIR); go test -v -tags=e2e -timeout=3h . -args -ginkgo.v -ginkgo.focus "functional tests" --managerImage $(CONTROLLER_IMG)-$(ARCH):$(TAG)
-
-.PHONY: test-conformance
-test-conformance: ## Run conformance test on workload cluster
-	PULL_POLICY=IfNotPresent $(MAKE) docker-build
-	cd $(TEST_E2E_DIR); go test -v -tags=e2e -timeout=4h . -args -ginkgo.v -ginkgo.focus "conformance tests" --managerImage $(CONTROLLER_IMG)-$(ARCH):$(TAG)
```

Comment:
i remove this target since i think it's unused

Reply:
Are we running the conformance tests as part of our E2E tests? I think we should do that, either as part of E2E tests or as a dedicated job.

Reply:
if you think we should do that, maybe running on dedicated job is better
i'll bring it back.

thanks

Reply:
i also restructure the code so it can be reusable when we want to write conformance test in the future